students/kpersic/module_3/blog/scss/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$primary:#006699;
```

Comment:
Razmisli o tome da stvoriš nekakvu konvenciju imenovanja varijabli. Na primjer, boje bi imale prefiks `color`, veličine prefiks `size`, i slično. Tako će ti biti lakše pronaći nešto u kodu (npr. tražiš boje i možeš tražiti `$color-` i dobiti ih sve), a isto tako će i osobe koje rade s tobom puno lakše znati koje varijable gdje koristiti (npr. neće se dogodi da netko stavi veličinu kao boju).

Postoje i pravila pisanja Sass koda koje je community raspisao kako bi se svi lakše držali najboljih praksi. Možeš ih pronaći ovdje: https://sass-guidelin.es/